src/components/InputWithLabel/InputWithLabel.module.css

```diff
@@ -0,0 +1,7 @@
+input[type="text"] {
```

Comment:
Is it the expected behavior that all input in the app will inherit this style? If yes, this is fine. If not. add a classname.

Reply:
Yes, I wanted to keep all input elements in the same style.